test/core/Safe.Execution.spec.ts

```diff
@@ -23,6 +23,7 @@ describe("Safe", () => {
                 function setStorage(bytes3 data) public {
                     bytes32 slot = 0x4242424242424242424242424242424242424242424242424242424242424242;
                     // solhint-disable-next-line no-inline-assembly
```

Comment:
This should use the `/* solhint-disable */ .. /* solhint-enable */` pattern right?

Reply:
Done 👍🏾